spec/feature/login_spec.rb

```diff
@@ -179,14 +179,14 @@
       end
     end
   end
-
   context "logging in from an OAuth journey" do
+    let(:application_scopes) { %i[openid] }
```

Comment:
There's no need for this to be in a `let` - the application can have the scopes `openid level0 level1`